src/imgTempGenerator.js

```diff
@@ -0,0 +1,37 @@
+const getImage = (value) => {
```

Comment:
- [ ] Since all the possible are mapped to the image file name, we can refactor this function to one liner as follows: 

```js
const getImage = (value) => `images/${value.toLowerCase()}.gif`;
```

Reply:
Thanks, changes added